code/modules/antagonists/blob/blob_minion.dm

```diff
@@ -0,0 +1,32 @@
+/datum/antagonist/blob_minion
+	name = "\improper Blob Minion"
+	antagpanel_category = ANTAG_GROUP_BIOHAZARDS
+	show_name_in_check_antagonists = TRUE
+	show_to_ghosts = TRUE
+	show_in_antagpanel = FALSE
+	/// The blob core that this minion is attached to
+	var/mob/camera/blob/overmind
+
+/datum/antagonist/blob_minion/New(mob/camera/blob/overmind)
+	. = ..()
+	src.overmind = overmind
+
+/datum/antagonist/blob_minion/on_gain()
+	forge_objectives()
+	. = ..()
+
+/datum/antagonist/blob_minion/greet()
+	. = ..()
+	owner.announce_objectives()
```

Comment:
does this make them talk in blobchat?

Reply:
no, i think blob chat is entirely handled in the mob

Reply:
ah owner isn't overmind, scuse meeeee